app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :order_decs, ->{order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
sự khác biệt của 2  lệnh này là gì?

Reply:
em nghĩ trong model thì user là đối tươngj chỉ cần có mặt để trong một record hợp lệ, không cần phải duy trì. Còn cái user_id cần phải duy trì cho tới khi validate xác nhận hợp lệ